data/remnant/remnant missions.txt

```diff
@@ -2231,3 +2243,678 @@ mission "Remnant: Face to Maw 2B"
+	on enter Nenia
+		dialog `You have never seen such activity from the void sprites. Many quickly congregate around the strange ship and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath Raider.`
+	on complete
+		event "remnant: nenia ember waste node"
+		conversation
+			`You follow the creature down through the clouds of Nasqueron and emerge into what can only be described as an amphitheater in the clouds. Thousands of void sprites fill the area, and the creature you have been following floats near the center of it. Your external sensors pick up myriad pulses of energy and eerie whistling noises that seem to resonate in the clouds. As the <ship> moves forward, there is a sudden silence, and the flickers of energy synchronize.`
+			`	You feel a sudden pressure in your mind and a sense of presence as the world around you fades out somewhat.`
+			`	"Know you not, do I. Live creature inside not-quite creature, like ones who flew but not everywhere." You are momentarily disoriented as you look around and see hundreds of views of yourself peering out the cockpit window. "Your intention reverberates - Know and not know - Return with more echoes - Perhaps I shall hear."`
+			`	With that, the pressure vanishes, and you are left staring out the window at the rapidly dispersing flock of void sprites. You aren't entirely sure what the voice meant, but you are certain that the Remnant are going to want to hear about this.`
+
+
+
+event "remnant: nenia ember waste node"
+	system "Nenia"
+		add fleet "Ember Waste Node" 10000
+	
+
+
+mission "Remnant: Cognizance 17"
+	name "Report back to Aventine."
+	description "The Remnant will want to know about what happened on Nasqueron."
+	source "Nasqueron"
+	landing
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 16: done"
+	
+
+mission "Remnant: Cognizance 18"
+	name "Go collect some void sprites"
+	description "Escort the Pelican fleet to <waypoints> and back to <planet>."
+	source
+		government "Remnant"
+	landing
+	destination Aventine
+	waypoint "Nenia"
+	to offer
+		has "Remnant: Cognizance 17: done"
+	on offer
+		conversation
+			`It is almost starting to feel commonplace to have Remnant pouncing on your ship and using your cockpit as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permission to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`
```

Comment:
```suggestion
			`It is almost starting to feel commonplace to have Remnant pouncing on your ship and using your bridge as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permission to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`
```

Then again I suppose "cockpit" would be more accurate if the player is flying a small interceptor type ship, but that's probably not very likely at this point in the storyline.

Reply:
Making a branch for this might be a possibility, but I'm not sure how important doing so would be.